app/api/backup/route.ts

```diff
@@ -20,13 +22,50 @@ export async function GET(request: NextRequest) {
     const currentTimeString = new Date(Date.now()).toJSON();
     console.log(`cron function triggered at ${currentTimeString}`);
 
+    let dbKeys = null;
+    try {
+        dbKeys = await getAllKeys("*");
+    } catch(error) {
+        console.error(`Error getting all the keys from the DB to backup: ${error}`);
+        return NextResponse.json("Internal Server Error", { status: 500 });
+    }
+
+    const keyPromises = dbKeys.map(k => getAndSaveKeyValue(k));
```

Comment:
> Does 'k' not have a type - @internette 

I mean it does... but I don't think we need to specify it because the function that is being called already type constrained it's incoming param.